### PR TITLE
Full group redesign

### PR DESCRIPTION
**Pull request by tonyjhuang**

### CODE REVIEW

**Comment by davlin93** [comment]
good shit man

File: css/pictures.css [hunk 1]
```diff
@@ -1,42 +1,70 @@
...
+  background: #EFEFEF;
+  text-align: middle;
+  width: 100px;
 }
 
 .group {
-  border: 1px solid black;
   height: 300px;
   text-align: center;
+  padding: 10px 5px !important;
```
Comment (davlin93) [inline css/pictures.css]: do we need this important?

File: js/main.js [hunk 2]
```diff
@@ -192,3 +184,78 @@ $(function() {
...
+    pictureHtml += _getPictureHtml(picture);
+  });
+  $groupElement.find(".pic-box").append(pictureHtml);
+
+
+
+  return $groupElement;
+}
+
+
```
Comment (davlin93) [inline js/main.js]: bonus lines

File: js/parse.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ var Parse = (function() {
       for(i = dataIndex + 1; i < rows.length; i++) {
         var cols = rows[i].split(",");
         var path = [picturePath, cols[0]].join("/");
-        pictures.push(new Picture(cols[1], path));
+        pictures.push(new Picture(parseInt(cols[1]), path));
```
Comment (davlin93) [inline js/parse.js:47]: my b

File: stimulus.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 
 <html>
 <head>
-  <title>Profile</title>
+  <title>Topkek</title>
```
Comment (davlin93) [inline stimulus.html:4]: pls

**Comment by davlin93** [comment]
+1

**Comment by tonyjhuang** [comment]
+1'ing for @yangk

**Pull request merged**